modules/amd-blocklisting-inbox-drivers-with-machineconfig.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * hardware_accelerators/amd-gpu-operator.adoc
+
+:_content-type: PROCEDURE
+[id="amd-blocklisting-inbox-drivers-with-machineconfig_{context}"]
+= Blocklisting the inbox drivers with a MachineConfig
+
+.Procedure
+
+. Verify that the inbox AMD drivers are loaded:
++
+[source,terminal]
+----
+$ oc debug node/worker0.example.com 
+----
++
+.Example output
+[source,terminal]
+----
+sh-4.4# chroot /host
+
+sh-5.1# lsmod | grep amd
+amdgpu               9494528  0
+iommu_v2               24576  1 amdgpu
+gpu_sched              49152  1 amdgpu
+drm_buddy              20480  1 amdgpu
+drm_display_helper    172032  1 amdgpu
+drm_ttm_helper         16384  1 amdgpu
+i2c_algo_bit           16384  2 mgag200,amdgpu
+ttm                    90112  2 amdgpu,drm_ttm_helper
+drm_kms_helper        192512  5 drm_display_helper,mgag200,amdgpu
+drm                   581632  10 gpu_sched,drm_kms_helper,drm_shmem_helper,drm_display_helper,mgag200,drm_buddy,amdgpu,drm_ttm_helper,ttm
+----
+
+. Verify that there is no existing `/etc/modprobe.d/amdgpu-blocklist.conf` file:
++
+[source,terminal]
+----
+sh-5.1# ls /etc/modprobe.d/amdgpu-blocklist.conf
+ls: cannot access '/etc/modprobe.d/amdgpu-blocklist.conf': No such file or directory
+
+sh-5.1# exit
+exit
+
+sh-4.4# exit
+exit
+
+Removing debug pod ...
+----
+
+. Create the `1-blocklist.yml` file. You should set `master` for the label `machineconfiguration.openshift.io/role` if you run Single Node OpenShift or `worker` in other scenarios with dedicated controllers.
++
+[source,terminal]
+----
```

Comment:
I think this can also be removed: it is part of blacklisting amd kernel module and should be covered by the AMD

Reply:
Blocklisting has been removed (see the Preview). I need to remove the file from the files view.